### PR TITLE
solution 80%

### PR DESCRIPTION
**Pull request by Yurko-Komanitskyi**
[DEMO LINK](https://yuron-maker.github.io/react_static-list-of-todos-js/)

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.jsx [hunk 1]
```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo'}
```
Comment (etojeDenys) [inline src/components/TodoInfo/TodoInfo.jsx:6]: let's use classNames package for such cases

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.jsx [hunk 2]
```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={classNames('TodoInfo', todo.completed
+    ? 'TodoInfo--completed'
+    : '')}
```
Comment (etojeDenys) [inline src/components/TodoInfo/TodoInfo.jsx:7]:
```suggestion
  <article className={classNames('TodoInfo', {'TodoInfo--completed' : todo.completed})}
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/components/TodoList/TodoList.jsx [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todoList }) => (
+  <section className="TodoList">
+    {todoList.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} />
+    ))}
+  </section>
```
Comment (VitaliyBodnar) [inline src/components/TodoList/TodoList.jsx:8]: please use semantic tags (`ul, li`) for this list

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
.

File: src/components/TodoList/TodoList.jsx (hunk 3 above)
Comment (yuron-maker) [reply src/components/TodoList/TodoList.jsx:8]: i.e. you need to write "ul" instead of a tag "section" and "li" instead of "article"?

**Pull request closed without merging**